Build.ps1

```diff
@@ -1,3 +1,10 @@
+[CmdletBinding()]
+param (
+    [Parameter(Mandatory = $false)]
+    [System.Boolean]
+    $RunTests = $false
```

Comment:
Usually in PowerShell you use `[switch]` for flags. Then run `./build.ps1 -RunTests` rather than `-RunTests $true`. Parameters also default to optional.